aat/src/resources/features/F-023/S-008.td.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Must return a negative response on malformed jurisdiction ID",
+  "_guid_": "S-008",
+  "_extends_": "F-023_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD - wants to see a list of valid documents they can print",
+    "contains a malformed jurisdictionID"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "jid": "$%£@^&*()_+",
+      "ctid": "FT_MasterCaseType",
+      "cid": "1111222233334444"
+    }
+    },
+
+  "expectedResponse": {
+    "responseCode": 400,
+    "responseMessage": "Bad Request",
```

Comment:
Please extent the common 400 response object. That will bring the 2 lines here and the headers below. (no need to then include the headers section at all.)

Reply:
done